src/lib/components/pagination/Pagination.js

```diff
@@ -0,0 +1,79 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+export default class Pagination extends Component {
+  constructor(props) {
+    super(props);
+
+    this.handleClick = this.handleClick.bind(this);
+    this.pageNumbers = this.pageNumbers.bind(this);
+    this.renderPages = this.renderPages.bind(this);
+    this.getPage = this.getPage.bind(this);
+    this.nextPage = this.nextPage.bind(this);
+    this.prevPage = this.prevPage.bind(this);
+  }
+
+  getPage(pageNumber) {
```

Comment:
changePage seems like more appropriate to me, I couldn't understand initially what this function will do